src/components/category/Category.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { categories } from "../../backend/db/categories";
+import "./category.css";
+
+
+export function Category() {
+  return (
+    <div className="categories">
+      <div className="category-title">Categories</div>
+      <div className="category-list">
+        {categories.map((category) => (
+          <div className="card">
```

Comment:
You can make a component card and return it with props.

Reply:
done